### DISCUSSION TITLE
Cause of the error during demo today

### DISCUSSION THREAD

**Discussion by jimhommes** [Spine-project/Spine-Toolbox#1485] [Support]
Hi all,

During the demo today during the webinar Juha encountered an error during the creation of the relationship parameters. I believe it was caused by Juha accidentally setting the alternative name first before the parameter name. I haven't tried it, just wanted to share this observation with you.

Great webinar!

**Comment by manuelma** [comment]
Thanks Jim
I was watching and I think the issue is about filtering. I'm guessing the parameter value was already there but Juha couldn't see it because of the filtering malfunctioning. That would explain why the app was refusing to add the parameter value?

I will try to reproduce it soon. Apart from that, I don't think the order in which the fields are entered matters, does it?

Bests,
Manuel

Télécharger Outlook pour Android<https://aka.ms/ghei36>
________________________________
From: jimhommes ***@***.***>
Sent: Wednesday, September 8, 2021 6:35:43 PM
To: Spine-project/Spine-Toolbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Spine-project/Spine-Toolbox] Cause of the error during demo today (#1485)


Hi all,

During the demo today during the webinar Juha encountered an error during the creation of the relationship parameters. I believe it was caused by Juha accidentally setting the alternative name first before the parameter name. I haven't tried it, just wanted to share this observation with you.

Great webinar!

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/Spine-project/Spine-Toolbox/discussions/1485>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ACXQ65BNNPDLH66RUJJI6VDUA6GF7ANCNFSM5DVHUHEA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
